### ISSUE TITLE
Documented issue title for adding a feedstock maintainer does not trigger the conda-forge-admin automation

### ISSUE THREAD

**Issue by kbrindley** [conda-forge/conda-forge-webservices#530]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list

The "updating the maintainer list" documentation describes adding an issue with the title

```
@conda-forge-admin, please add user @username
```

to trigger an automated process that updates the maintainer list. However, this title doesn't trigger the automated process. The workaround is to delete the comma as

```
@conda-forge-admin please add user @username
```

which then does trigger the automated process.

* Behavior example seen here: https://github.com/conda-forge/waves-feedstock/issues/2
* Originally reported as a documentation update here: https://github.com/conda-forge/conda-forge.github.io/pull/1892

### Installed packages

```shell
Not applicable
```

### Environment info

```shell
Not applicable
```

**Comment by kbrindley** [comment]
Existing tests appear to catch the existing documentation example

```
@conda-forge-admin, please add user @username
```

https://github.com/conda-forge/conda-forge-webservices/blob/main/conda_forge_webservices/tests/test_commands.py#L271
```
'@conda-forge-admin, please add user @blah',
```

Possibly related to the user name in the linked behavior example, which had mixed case and a hyphen? Perhaps the user is captured correctly when the comma is excluded?

**Comment by kbrindley** [comment]
I can't run the tests locally. Poking around the ``commands.py`` source, I think I can re-create the RegEx. I don't think my observed behavior is a RegEx issue. This is probably about as far as I can chase this issue.

```
$ cat test.py
import re

pre = r"@conda-forge-(admin|linter)\s*[,:]?\s*"
ADD_USER = re.compile(pre + r"(please )?add user @(?P<user>\S+)$", re.I)

tests = [
    "@conda-forge-admin, please add user @My-NAME",
    "@conda-forge-admin please add user @My-NAME"
]

for text in tests:
    match = ADD_USER.search(text)
    print(match.group('user'))
$ python test.py
My-NAME
My-NAME
```

**Comment by beckermr** [comment]
This appears to have fixed itself and IDK why. Let's close for now.

**Comment by jakirkham** [comment]
Saw this again with issue ( https://github.com/conda-forge/benchling-sdk-feedstock/issues/11 ). Have reopened to track

**Comment by beckermr** [comment]
Yeah this is weird. I tested extensively while watching the webserver logs and could not reproduce. Arghhhh. Thanks for reopening.

**Comment by jakirkham** [comment]
No worries. It's an odd issue. Don't recall there being lots of changes to relevant code paths here, which makes it a bit confusing

Is it possible there's an issue on GitHub's end somehow? TBH I don't know how this would happen. Just trying to come up with other ideas

**Comment by beckermr** [comment]
That is possible. It's hard, we if we can find the webhook event in the settings pane for conda-forge, it's possible to resend it to debug love.

**Comment by jakirkham** [comment]
Adding another data point, saw this with re-rendering issue request recently ( https://github.com/conda-forge/shellcheck-feedstock/issues/12 )

**Comment by ap--** [comment]
I think this might have the same issue: conda-forge/universal_pathlib-feedstock#26

**Comment by jakirkham** [comment]
Yeah try deleting the `,`

**Comment by jakirkham** [comment]
Maybe

Went ahead and tried including the `<space>` to start and it seemed to work ( https://github.com/conda-forge/openmpi-feedstock/issues/120 )

Suppose it could just be luck though. We could try more tests

**Comment by jaimergp** [comment]
One thing I noticed while debugging the 🚀 reacts to the commands is that the new-issue event payload is sent before the API can return the issue content. So we need to re-run the issue fetching logic a in a tiny loop. This is done in one more part in the code, IIRC. See:

https://github.com/conda-forge/conda-forge-webservices/blob/9978abd1e8b84fda67cbb65b37f8143489a23a86/conda_forge_webservices/commands.py#L108-L121

And:

https://github.com/conda-forge/conda-forge-webservices/blob/9978abd1e8b84fda67cbb65b37f8143489a23a86/conda_forge_webservices/commands.py#L356-L371

**Comment by beckermr** [comment]
Yes a race condition makes sense.
